impl/src/main/java/org/jboss/cdi/tck/tests/full/deployment/discovery/VerifyingExtension.java

```diff
@@ -14,7 +14,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.jboss.cdi.tck.tests.deployment.exclude;
+package org.jboss.cdi.tck.tests.full.deployment.discovery;
```

Comment:
This is a Git hiccup I guess?

Reply:
Yep, I certainly didn't do any such wild renames :-)